docs/wallet-integration.md

```diff
@@ -0,0 +1,126 @@
+## Wallet Integration
+
+This guide explains how to integrate wallets with the Babylon staking dApp. The
+dApp supports both Bitcoin (BTC) and Babylon (BBN) wallets through two
+integration paths:
+
+### 1. Integration Through Tomo Connect SDK Lite (Recommended)
+
+The recommended way to integrate your wallet with Babylon staking dApp is
+through
+[Tomo Connect SDK Lite](https://docs.tomo.inc/tomo-sdk/tomo-connect-sdk-lite).
+This provides:
+
+- Simplified integration process
+- Broader wallet ecosystem access
+- Maintained by Tomo team
+
+We strongly encourage wallet developers to integrate via Tomo Connect first.
+Please refer to Tomo's documentation for integration details.
+
+### 2. Direct Native Integration
+
+Direct native integration should only be considered if Tomo Connect integration
+is not feasible for your use case. This repository contains a set of natively
```

Comment:
Direct native integration happens based on our choice, not if it is feasible for the third party wallet's use case or not.